src/commit-split.ts

```diff
@@ -71,16 +66,15 @@ export class CommitSplit {
       const dedupe: Set<string> = new Set();
       for (let i = 0; i < commit.files.length; i++) {
         const file: string = commit.files[i];
-        const path = relative(this.root, file);
```

Comment:
@bcoe I believe this usage of `path.relative` was automatically converting path separators. I still don't have access to a windows machine with node to verify but my manifest tests all pass with this change.

Reply:
In fact I think we should probably have a lint check to prohibit any `"path"` module usage under `src/` as it will do the wrong thing under windows (since github api always returns `/` separators)